CityOfDetroit/bloom
# feat: show a11y features on public site

## Issue

- Closes #975 

## Description

(1) Show accessibility features in a bulleted list on the public site under the features category
(2) Show a tag on the image card that the listing is accessible if it has at least one accessibility feature

## Type of change


## How Can This Be Tested/Reviewed?

Add and remove a11y features from a listing in partners and ensure the bullet points and tags come through

## Checklist:
